enhancements/pac-cel-filtering.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Option to disable default creation of service account and rbac resources
+authors:
+  - "@chmouelito"
+creation-date: 2021-03-14
+status: implementable
+---
+
+# TEP-XXX: Pipelines as code event filtering with CEL
+
+## Summary
+
+This proposal helps user to do advanced filtering for PipelineRun selections.
+
+## Motivation
+
+We currently have a very basic filtering mechanism for Pac PipelineRuns.
```

Comment:
you mean like fetching the actual user pipelineRun by labels right?
using kubectl
with pac labels